packages/2019-disaster-game/src/constants/env.js

```diff
@@ -0,0 +1,18 @@
+// Static site configuration as set in the environment of the server
+// TODO: This is currently written under the assumption that this module
+// will be lazy-evaluated after the DOM is ready. This isn't necessarily
+// true and could lead to some bewildering debugging.
```

Comment:
Looks like this isn't quite addressed? If this isn't an issue, maybe remove the comment?

Reply:
It's not an issue _today_. My concern is that something will change in the future that will cause this to break. In the event that that happens, maybe the comment is still too little too late?